Automattic/wp-calypso
# Themes showcase: remove Beta badge for FSE themes

#### Changes proposed in this Pull Request

* Remove the "Beta" badge for block themes on FSE eligible sites
* Remove the "Full Site Editing" theme tab, since the themes are the same as what we show in the Recommended tab

Now that we are rapidly rolling out FSE to new users through the /start flow without any messaging about it being in beta, it seems better not to have "Beta" plastered all over the themes showcase.

#### Testing instructions

* On an FSE eligible site (you can create one through the wordpress.com/new flow, or horizon.wordpress.com/start)
* See that the themes showcase no longer shows the Beta badge on FSE themes
* And that the "Full Site Editing" theme tab is gone.

| **Before** | **After** |
| - | - |
| <img width="718" alt="image" src="https://user-images.githubusercontent.com/1699996/151635716-dab8efeb-d129-4db9-848d-183238e0ff1e.png"> | <img width="702" alt="image" src="https://user-images.githubusercontent.com/1699996/151635746-bedcb6e3-9f54-4ba1-a085-6f591c9557e6.png"> |



Resolves https://github.com/Automattic/wp-calypso/issues/58930
